README.md

```diff
@@ -1,73 +1,97 @@
-[up-for-grabs.net](https://up-for-grabs.net/)
-![Continuous Integration status](https://github.com/up-for-grabs/up-for-grabs.net/workflows/Continuous%20Integration/badge.svg)
+# Up-For-Grabs
+
+[![Continuous Integration status](https://github.com/up-for-grabs/up-for-grabs.net/workflows/Continuous%20Integration/badge.svg)](https://github.com/up-for-grabs/up-for-grabs.net/actions)
 [![Netlify Status](https://api.netlify.com/api/v1/badges/0ee7bf9f-1aed-465b-8e9e-01de009a8a7e/deploy-status)](https://app.netlify.com/sites/up-for-grabs-test-bench/deploys)
 [![All Contributors](https://img.shields.io/badge/all_contributors-11-orange.svg?style=flat-square)](#contributors)
-================
-
-<a href="https://up-for-grabs.net/"><img width="814" src="https://user-images.githubusercontent.com/359239/67390578-50f83a00-f573-11e9-835d-02eb9e019afb.png"></a>
-
-This repository contains the content for the [Up-For-Grabs website](https://up-for-grabs.net/), a list of projects with curated tasks for new contributors.
-
-## List your project
-
-If you know of or own a project that should be listed on Up for Grabs,
-[follow the instructions to open a pull request](docs/list-a-project.md).
-
-## Contributing
-
-If you would like to get involved with the project, please read the
-[CONTRIBUTING.md](.github/CONTRIBUTING.md) file as it contains:
-
- - Instructions about getting your environment setup
- - Guidance for testing locally
- - Commands to run to verify changes
-
-## How does the site work?
-
-If you want to learn more about the various parts of the Up-for-Grabs project,
-the [How it works](docs/how-it-works.md) document is a good overview.
-
-## Contributors ✨
```

Comment:
The fact that this change removes the CONTRIBUTORS section, which is auto-generated, makes this a non-starter for me.

If you want to write a better CONTRIBUTING guide in a new file, there is already a [`.github/CONTRIBUTING.md`](https://github.com/up-for-grabs/up-for-grabs.net/blob/gh-pages/.github/CONTRIBUTING.md) file available.

There are also other docs under `docs/` which could likely be improved.